### DISCUSSION TITLE
OOM for TrailO ?

### DISCUSSION THREAD

**Discussion by Mikke73Swe** [OpenOrienteering/mapper#1972] [Ideas]
Couldn't find better way to ask ;-(

Is there any plans for develope OOM better for using when making TrailO courses ?

Thinking about making course design and control description sheets. (all special control descriptions.... A-Z etc)
Also making the "solution maps" and "timed controll" (also TempO) in easy way.

**Comment by ghost** [comment]
> Thinking about making course design and control description sheets.

As for making course design & control description sheets, you could do that *manually* using built-in "Course Design" symbol set:

- https://www.openorienteering.org/mapper-manual/pages/course_design.html

**Comment by dg0yt** [comment]
> Is there any plans for develope OOM better for using when making TrailO courses ?

There were e-mail requests for TrailO support but there is no tracking ticket. Given the backlog of requested features it won't happen soon, unless someone steps up to work on it as a contributor.
